app/src/main/java/com/github/polypoly/app/base/game/Game.kt

```diff
@@ -32,6 +30,8 @@ class Game private constructor(
     val dateBegin: Long = System.currentTimeMillis(),
     val inGameLocations: List<InGameLocation> = rules.gameMap
         .flatMap { zone -> zone.locationProperties.map { InGameLocation(it) } },
+    val transactions : ArrayList<GameTransaction> = arrayListOf(),
+    val pastTransactions: ArrayList<List<GameTransaction>> = arrayListOf(),
```

Comment:
Maybe you could have a `Map<Int, List<GameTransaction>>` instead to better express that each list is for one turn (and make it more easily usable if needed)

Reply:
thats right :)!